src/esw/brushless.py

```diff
@@ -244,37 +381,45 @@ def __init__(self):
         _ratio_motor_to_wheel = rospy.get_param("wheel/gear_ratio")
         self.WHEELS_M_S_TO_MOTOR_RAD_RATIO = (1 / rospy.get_param("wheel/radius")) * rospy.get_param("wheel/gear_ratio")
 
-        _max_speed_m_s = rospy.get_param("rover/max_speed")
-        assert _max_speed_m_s > 0, "rover/max_speed config must be greater than 0"
+        self.max_motor_rps = rospy.get_param("brushless/drive/max_motor_rps")
+        self._max_motor_speed_rad_s = self.max_motor_rps * 2 * math.pi
+
+        drive_controller_info_by_name = rospy.get_param("brushless/drive/controllers")
+        self._drive_names = []
+        self._drive_command_data_list = []
+        for name, info in drive_controller_info_by_name.items():
+            self._drive_names.append(name)
+            self._drive_command_data_list.append(
+                CommandData(
+                    CommandData.POSITION_FOR_VELOCITY_CONTROL, CommandData.ZERO_VELOCITY, CommandData.DEFAULT_TORQUE
+                )
+            )
 
-        max_motor_rad_s = _max_speed_m_s * self.WHEELS_M_S_TO_MOTOR_RAD_RATIO
-        self.MEASURED_MAX_MOTOR_RAD_S = 1 * 2 * math.pi  # Should not be changed. Derived from testing.
-        self._max_motor_speed_rad_s = min(self.MEASURED_MAX_MOTOR_RAD_S, max_motor_rad_s)
-        self._last_updated_time = t.time()
+        using_pi3_hat = rospy.get_param("brushless/using_pi3_hat")
+        if using_pi3_hat:
+            pass
+            # import moteus_pi3hat
+            # transport = moteus_pi3hat.Pi3HatRouter(
+            #     servo_bus_map={
+            #         1: [11],
+            #         2: [12],
+            #         3: [13],
+            #         4: [14],
+            #     },
+            # )
```

Comment:
Shouldn't we at least say transport = None? Or just get rid of this if-statement?

Reply:
same with in the arm controller

Reply:
yes good catch